tests/testSchema.py

```diff
@@ -132,11 +178,11 @@ def testKeyAccessors(self):
 
     def testComparison(self):
         schema1 = lsst.afw.table.Schema()
-        schema1.addField("a", type=float, doc="doc for a", units="units for a")
-        schema1.addField("b", type=int, doc="doc for b", units="units for b")
+        schema1.addField("a", type=float, doc="doc for a", units="")
+        schema1.addField("b", type=int, doc="doc for b", units="")
         schema2 = lsst.afw.table.Schema()
-        schema2.addField("a", type=int, doc="doc for a", units="units for a")
-        schema2.addField("b", type=float, doc="doc for b", units="units for b")
+        schema2.addField("a", type=int, doc="doc for a", units="")
+        schema2.addField("b", type=float, doc="doc for b", units="")
```

Comment:
To maintain full test coverage here, I think we want to continue to pass real units for all of these calls (it doesn't matter what they are , but it matters that "a" and "b" have different units).